### DISCUSSION TITLE
Can my pyo3/maturin library be added to the examples section?

### DISCUSSION THREAD

**Discussion by jessekrubin** [PyO3/pyo3#3527] [Questions]
It is a nearly drop-in replacement for the mapbox-mercantile library: https://github.com/jessekrubin/utiles

PS: yall nailed PyO3!

**Comment by davidhewitt** [comment]
Sure thing, please open a PR. Great to hear that you enjoyed using PyO3!

**Comment by jessekrubin** [comment]
@davidhewitt boom https://github.com/PyO3/pyo3/pull/3529

**Comment by jessekrubin** [comment]
@davidhewitt do I go ahead and hit "queue pr"? What is the repo etiquette?

**Reply by adamreichold** [reply]
You should not be able to do that. Someone of the maintainers will pick up the PR and review. If there is feedback you should take care of that until you get an approval. Then usually the maintainer who approved it will merge the PR. (Since the change is simple, this should be fast, but nobody found the time yet. Will do so now...)
